docs/implementation/IMPROVEMENTS_PLAN.md

```
@@ -374,7 +374,7 @@ dependencies = [
 - [ ] `test_state_schema.py` - processed.json validation
 - [ ] `test_schema_migration.py` - Version upgrades
 
-[↑ Back to top](#mamfast-improvements-plan)
+[↑ Back to top](#Shelfr-improvements-plan)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid
Expected: #shelfr-improvements-plan; Actual: #Shelfr-improvements-plan

(MD051, link-fragments)